pagopa/io-app
# chore: [IOAPPCOM-7] Replaces tabs with Navigation top tabs

## Short description
This PR introduces navigation tabs in places of static tab with native-base or specific packages on services and messages home

## How to test
Navigate through messages and services home to see no specific feature regressions

![image](https://user-images.githubusercontent.com/3959405/203297607-783b4225-7403-48a7-8551-a89a379f5f28.png)
In the image on the left is the incoming UI and on the right is the as-is UI
